### PR TITLE
LKE-4395 Properties order

### PR DESCRIPTION
**Pull request by Diachk**
https://linkurious.atlassian.net/browse/LKE-4395

### CODE REVIEW

File: src/api/DataSource/types.ts [hunk 1]
```diff
@@ -86,6 +87,7 @@ export interface DataSourceUserInfo {
 export interface ISetDefaultSourceStylesParams extends IDataSourceParams {
   styles?: IDataSourceDefaultStyles;
   captions?: ICaptions;
+  propertiesOrder?: IPropertiesOrder;
```
Comment (aminerhanemi) [inline src/api/DataSource/types.ts:90]: why propertiesOrder is needed here ?

Comment (Diachk) [reply src/api/DataSource/types.ts:90]: Isn't this how you set it from the UI using the setDefaultsAPI ?

Comment (aminerhanemi) [reply src/api/DataSource/types.ts:90]: you're right just the API name is confusing "setDefaultSourceStyles".
Now it is used for style caption and properties order

Comment (Diachk) [reply src/api/DataSource/types.ts:90]: You want to rename it to just `setDefaults` ?

File: src/api/DataSource/types.ts [hunk 2]
```diff
@@ -78,6 +78,7 @@ export interface DataSourceUserInfo {
   features: DataSourceFeatures;
   defaultStyles?: IDataSourceDefaultStyles; // defined if withStyles or withCaptions was set to true in the request and the data-source is connected
   defaultCaptions?: ICaptions;
+  defaultPropertiesOrder?: IPropertiesOrder;
```
Comment (francescoinfante) [inline src/api/DataSource/types.ts:81]: when is it undefined?
what about "withStyles" and "withCaptions" params in IGetDataSourcesStatusParams interface?

**Review by francescoinfante — CHANGES_REQUESTED** [review]

File: src/api/DataSource/types.ts (hunk 2 above)
Comment (Diachk) [reply src/api/DataSource/types.ts:81]: It is defined when the properties order has been configured and undefined otherwise.

Comment (francescoinfante) [reply src/api/DataSource/types.ts:81]: why not always defined with default to empty object?
why not keeping consistent with the other styles?

Comment (Diachk) [reply src/api/DataSource/types.ts]: It is simpler to keep it defined with a default empty object

**Review by francescoinfante — APPROVED** [review]

**Pull request merged**